### DISCUSSION TITLE
Difference between Set “Language-Team” header and Contributors in comment add-on

### DISCUSSION THREAD

**Discussion by m-aciek** [WeblateOrg/weblate#9448] [Q&A]
Hello, what's the difference between ["Set Language-Team header"](https://docs.weblate.org/en/weblate-4.18.1/admin/projects.html#project-set-language-team) setting (project level) and ["Contributors in comment"](https://docs.weblate.org/en/weblate-4.18.1/admin/addons.html#contributors-in-comment) add-on? The features seem to be very similar. Currently the documentation is laconic about them. Does one require the other? I would be willing to post a PR with some documentation extension in this matter.

**Comment by m-aciek** [comment]
I realized that those are different metadata fields, Language-Team and Translators. Closing now.

**Comment by nijel** [comment]
Yes, it is stored differently. Language team setting is at the component level for legacy reasons – it was introduced at times when Weblate didn't have ambitions to support that many formats. New options are introduced via add-ons until https://github.com/WeblateOrg/weblate/issues/1897 is implemented.
